.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/blinkist/blinkist-config
+    parallelism: 1
+    docker:
+    - image: circleci/ruby:2.3.6-node-browsers
+    steps:
+    - checkout
+    - run: 
+        name: Bundle Install
```

Comment:
https://github.com/blinkist/blinkist-core/blob/master/.circleci/config.yml#L20 before this guy?